clang/test/CheckedC/inferred-bounds/equiv-expr-sets.c

```diff
@@ -1436,21 +1447,21 @@ void original_value19(array_ptr<int> *x, array_ptr<int> *i) {
   // CHECK-NEXT: }
   // CHECK-NEXT: }
 
-  // Original value of i in i - 1: x
-  // Updated EquivExprs: { { x - 1, i } }, Updated SameValue: { i + 1 }
+  // Original value of i in i - 1: i + 1
+  // Updated EquivExprs: { { i + 1, i } }, Updated SameValue: { i + 1 }
```

Comment:
Looks like there is a typo here: the updated equivalent expressions must be `{i+1, x}`.